testing/trino-product-tests/src/main/resources/io/trino/tests/hive/data/all_types/openx.json

```diff
@@ -0,0 +1 @@
+{c_tinyint:127,c_smallint:32767,c_int:2147483647,c_bigint:9223372036854775807,c_float:123.345,c_double:234.567,c_decimal:345.678,c_decimal_short:345.678,c_decimal_full:12345678901234567890.123456789012345678,c_timestamp:"2015-05-10 12:15:35.123456",c_date:"2015-05-10",c_string:"ala ma kota",c_varchar:"ala ma kot",c_char:"ala ma    ",c_boolean:true,c_binary:"a290IGJpbmFybnk="}
```

Comment:
FYI the serde has some "features" which might also need testing

https://github.com/rcongiu/Hive-JSON-Serde#importing-malformed-data and the others in the list.

Reply:
Thanks, I didn't notice that I forgot the quotes. My built-in parser also does auto-correct 😄